src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -130,6 +130,11 @@ export interface ConnectPeerArgs {
   host: string;
 }
 
+export interface TlvRecord {
```

Comment:
I don't think we should add this here - all the other connectors and internally the browser extension uses a different format for TLV records